MergeSharpWebAPI/ServerConnection/RetryPolicy.cs

```diff
@@ -0,0 +1,10 @@
+using Microsoft.AspNetCore.SignalR.Client;
+namespace MergeSharpWebAPI;
```

Comment:
```suggestion
namespace MergeSharpWebAPI.ServerConnection;
```

Reply:
i'd apply this myself but idk if it messes up the using's in other files n im too lazy to checkout n build myself